infra/config/validation.yaml

```
@@ -3,70 +3,82 @@ allowed_keywords:
     requires_scope: true
     scopes: [ Server, Config ]
-    example: "feat(Config): Add dynamic environment variable loading"
+    examples:
+      - "feat(Config): Add dynamic environment variable loading"
+
   - name: fix
     description: Fix a bug.
     requires_scope: true
-    scopes: [ Database ]
-    example: "fix(Database): Resolve connection timeout issue"
+    scopes: [ Database, Code ]
```

YAMLlint (1.35.1)

[error] too many spaces inside brackets

(brackets)